includes/admin/plugins.php

```diff
@@ -325,6 +324,20 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			'<button type="button" class="button button-disabled" disabled="disabled">%s</button>',
 			esc_html( _x( 'Active', 'plugin', 'default' ) )
 		);
+		$all_plugins    = array_keys( get_plugins() );
+		$plugin_slug    = $plugin_data['slug'];
+		$plugin_path    = '';
+		foreach ( $all_plugins as $path ) {
+			if ( strpos( $path, $plugin_slug ) !== false ) {
```

Comment:
Safer to do this:

```suggestion
			if ( strtok( $file, '/' ) === $plugin_slug ) {
```

This is because the multiple plugins may use the slug as a prefix, like `woocommerce` as well as `woocommerce-gateway-stripe`.

Instead of `strtok()` this could also use `dirname()`.